### DISCUSSION TITLE
Logs: Better separation of timestamp and log line

### DISCUSSION THREAD

**Discussion by svennergr** [grafana/grafana#64390] [Dashboards, Panels & Visualizations]
![image](https://user-images.githubusercontent.com/8092184/223697110-c74bc05c-b499-44fc-b527-c01c6739e8a6.png)

Log lines and timestamps are currently displayed in the same way, with just a little space in between them. That might make it hard to read and focus on one of the two if there are many loglines.

Let's use this issue to discuss possible scenarios how to solve this and create a better separation of timestamp and logline.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82795. Feel free to continue the discussion around this there. Thank you!
